packages/wrangler/src/docs/index.ts

```diff
@@ -1,55 +1,49 @@
-import { printWranglerBanner } from "..";
-import { readConfig } from "../config";
+import { defineCommand } from "../core";
 import { logger } from "../logger";
-import * as metrics from "../metrics";
 import openInBrowser from "../open-in-browser";
 import { runSearch } from "./helpers";
-import type {
-	CommonYargsArgv,
-	StrictYargsOptionsToInterface,
-} from "../yargs-types";
 
-export function docsOptions(yargs: CommonYargsArgv) {
-	return yargs
-		.positional("search", {
+defineCommand({
+	command: "wrangler docs",
+	metadata: {
+		description: "📚 Open Wrangler's command documentation in your browser\n",
```

Comment:
Do you know what is the usage of `\n` here? Is it used to adjust the layout of the help message?

Reply:
yeah, there are line breaks to group the commands in the help